### DISCUSSION TITLE
selecting long line as multiple lines

### DISCUSSION THREAD

**Discussion by mauricioscastro** [Eugeny/tabby#2750] [General]
hello and thank you for this great terminal.

is there a config that will select a long line and then pasting them without breaking them as multiple lines?

I have this behavior that sometimes confuses me.

best regards.

**Comment by poudenes** [comment]
Ohh I have same question. Now a long token will be on multiple lines. Disable "Line break" will be great

**Reply by R-lmmortal** [reply]
How do I disable line breaks? I want the log output to be displayed on one line instead of multiple lines.
